pkgs/development/compilers/llvm/6/clang/default.nix

```diff
@@ -51,13 +51,17 @@ let
     outputs = [ "out" "lib" "python" ];
 
     # Clang expects to find LLVMgold in its own prefix
-    # Clang expects to find sanitizer libraries in its own prefix
+    # Clang expects to find sanitizer libraries and headers in its own prefix
     postInstall = ''
       if [ -e ${llvm}/lib/LLVMgold.so ]; then
         ln -sv ${llvm}/lib/LLVMgold.so $out/lib
       fi
 
-      ln -sv ${llvm}/lib/clang/${release_version}/lib $out/lib/clang/${release_version}/
```

Comment:
Can we use a `-B` flag in libcxxstdenv instead? That is good for parallelism

Reply:
I don't follow. Use `-B` where to do what?

Reply:
Eh, I'll just not worry about it for now.